install/app-neovim.sh

```diff
@@ -4,8 +4,10 @@ sudo apt install -y neovim
 
 if [ ! -d "$HOME/.config/nvim" ]; then
 	git clone https://github.com/LazyVim/starter ~/.config/nvim
+	mkdir -p ~/.config/nvim/lua/plugin/after
+	cp ~/.local/share/omakub/configs/neovim/transparency.lua ~/.config/nvim/plugin/after/
```

Comment:
Why this new directory?

Reply:
This will allow neovim to load the transparency configs at the end of other configs, making sure all other highlights are loaded first